README.md

```diff
@@ -50,14 +50,24 @@ On Android the View state is not persisted consistently across Activity restarts
 
 For most people using an app built from the react-native template, that means editing `MainActivity.java`, likely located in `android/app/src/main/java/<your package name>/MainActivity.java`
 
-You should add this code, which specifically discards any Activity state persisted during the Activity restart process, to avoid inconsistencies that lead to crashes.
+You should add this code, which specifically discards any Activity state persisted during the Activity restart process, to avoid inconsistencies that lead to crashes. You should place this in the parent MainActivity and not inside the MainActivityDelegate.
```

Comment:
```suggestion
You should add this code, which specifically discards any Activity state persisted during the Activity restart process, to avoid inconsistencies that lead to crashes.
Please note that the override code should not be placed inside `MainActivityDelegate`, but rather directly in `MainActivity`.
```

Reply:
Just a rephrase request